### DISCUSSION TITLE
I can't find the Keyboard Layout New AZERTY - AFNOR

### DISCUSSION THREAD

**Discussion by Neocamelus** [linuxdeepin/developer-center#3449] [Q&A | 问答板块]
I have a French AZERTY keyboard 105 keys With Euro.
It is a keyboard that respects the new Afnor AZERTY standard ( [NF Z71-300](https://norme-azerty.fr/en/) )

I would like to use my keyboard with Deepin 20.7 but only the traditional Azerty is offered.

Here is its configuration in XFree86 under Debian: 
X11 Layout: fr
X11 Model: pc105
X11 Variant: afnor

How could I use my keyboard with Deepin?

**Comment by BLumia** [comment]
cc @tsic404

**Comment by Neocamelus** [comment]
Please, add the FRENCH AFNOR Standardized AZERTY layout which is present in the XKB-DATA package in Debian.
